user_guide.adoc

```diff
@@ -2,8 +2,7 @@
 Kaitai Project
 v0.8
 :toc: left
-:source-highlighter: coderay
-:numbered:
+:source-highlighter: pygments
```

Comment:
If this is not needed in all files, then probably we should not add it here. If it is actually required, then we should replace `:source-highlighter: coderay` in all files with this, not just delete it.

Reply:
Good catch, it's my mistake, sorry. Of course it's not needed, because I set the `source-highlighter` in the Makefile.